src/condor_schedd.V6/schedd.cpp

```diff
@@ -1667,12 +1667,10 @@ Scheduler::count_jobs()
 		} else if ( current_time - owner_info.LastHitTime > AbsentOwnerLifetime ) {
 			// mark user creds for sweeping.
 			if (cred_dir_krb) {
-				dprintf(D_FULLDEBUG, "creating credmon KRB mark file for user %s\n", owner_info.Name());
-				credmon_mark_creds_for_sweeping(cred_dir_krb, owner_info.Name());
+				credmon_mark_creds_for_sweeping(cred_dir_krb, owner_info.Name(), credmon_type_KRB);
 			}
 			if (cred_dir_oauth) {
-				dprintf(D_FULLDEBUG, "creating credmon OAUTH mark file for user %s\n", owner_info.Name());
```

Comment:
Can there still be some logging made available for when the `count_jobs` timer thinks a mark file should be written (either here or in `credmon_mark_creds_for_sweeping`)?

Reply:
I didn't want to log the message in the case where we decide there's nothing to clean up. But I can add the message inside credmon_mark_creds_for_sweeping() after that check.

Reply:
Thanks!